hal/lib/shared/HAL.cpp

```diff
@@ -18,17 +18,22 @@ struct HAL_JoystickAxesInt {
 
 extern "C" {
 
-int HAL_GetControlWord(HAL_ControlWord* data) {
-  return FRC_NetworkCommunication_getControlWord((ControlWord_t*)data);
+HAL_ControlWord HAL_GetControlWord() {
```

Comment:
The main motivation behind this was to avoid parsing pointer argument outputs. If we have that handled, I can revert this. We shouldn't be placing ease of generation over performance anyway.